DataSourceTests/QuickSpecWithDataSets.swift

```diff
@@ -44,4 +45,11 @@ class QuickSpecWithDataSets: QuickSpec {
 
 	let dataSetWithTestCellModels = [TestCellModel(), TestCellModel(), TestCellModel()]
 	let dataSetWithTestCellModels2 = [TestCellModel()]
+
+	var coreDataManager: CoreDataManager?
```

Comment:
This probably should be an implicitely unwrapped optional:
```suggestion
	var coreDataManager: CoreDataManager!
```